projectile.el

```diff
@@ -1480,6 +1481,7 @@ With a prefix ARG invalidates the cache first."
 (defvar projectile-haskell-cabal '("*.cabal"))
 (defvar projectile-rust-cargo '("Cargo.toml"))
 (defvar projectile-r '("DESCRIPTION"))
+(defvar projectile-npm '("package.json" "node_modules"))
```

Comment:
Will this look for `package.json` *and* a `node_modules` dir? Your PR message seems to indicate that, if so I think it should rather be an *or* instead, since if you clone an npm module from, say, GitHub it won't have a node_modules directory until you issue an `npm install`.